raiden/network/rpc/client.py

```diff
@@ -216,6 +216,13 @@ def estimate_and_transact(classobject, callobj, *args):
     return transaction_hash
 
 
+def get_block_gaslimit(client):
```

Comment:
Perhaps it's not the best idea to have an extra RPC call each time we want to make a transaction. Instead I would advise to set this each time the [callback of the alarm task for a new blocknumber](https://github.com/raiden-network/raiden/blob/master/raiden/raiden_service.py#L131) is executed.

Or have a new task that runs every `X` mins and queries the current block and gets its gas limit.